api/specs.proto

```diff
@@ -274,6 +274,9 @@ message ClusterSpec {
 
 	// TaskDefaults specifies the default values to use for task creation.
 	TaskDefaults task_defaults = 7 [(gogoproto.nullable) = false];
+
+	// LockKey is the encryption key for the raft data
```

Comment:
Also, do we need to store the new and old key here in case the migration is interrupted?